examples/api-routes/api/test.post.ts

```diff
@@ -1 +1,28 @@
+import { defineOpenAPISchema } from "#internal/nitro/routes/openapi";
+
 export default defineEventHandler(() => "Test post handler");
+
+defineOpenAPISchema({
+    method: 'post',
+    routeBase: '/api/test',
```

Comment:
Some props such as method, and url, can be auto-generated already without user needing them